src/command_modules/azure-cli-managedservices/azure/cli/command_modules/managedservices/__init__.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.command_modules.managedservices._help import helps  # pylint: disable=unused-import
+from ._command_type import ManagedServicesCommandGroup
+
+
+class ManagedServicesCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        managedservices_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.managedservices.custom#{}')
+        super(ManagedServicesCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                            min_profile='2017-03-10-profile',
```

Comment:
@limingu please provide guidance on the right way to annotate this. `min_profile` has been removed.

Reply:
Remove the line of min_profile='2017-03-10-profile',

Reply:
Create ResourceType for ManagedServices such as MGMT_MANAGEDSERVICES and add as resource_type=ResourceType.MGMT_MANAGEDSERVICES ,
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L33